.gitignore

```diff
@@ -75,6 +75,7 @@ docs/manual/figures/signature-types-with-canonicalname.dia
 docs/manual/*.png
 docs/manual/*.svg
 docs/examples/**/*.class
+docs/manual/manual.html-e
```

Comment:
Every time that a run `make`, `manual.html-e` file is created, but I couldn't identify which specific part creates it. Perhaps this file is only created on Mac OS. Should I remove this line from `.gitignore`?